app/controllers/merchant/items_controller.rb

```diff
@@ -1,4 +1,22 @@
 class Merchant::ItemsController < ApplicationController
   def index
+    if params[:merchant_id]
+      @merchant = Merchant.find(params[:merchant_id])
+      @items = @merchant.items
+    else
+      @items = Item.all
+    end
+  end
+
+  def update_status
```

Comment:
@Jonathan-M-Wilson `def update`

Reply:
In keeping with my comment on the routes file, this would serve as a general purpose `update` path for items (through merchant). Later, when we are editing an item, we can specify what is happening in this method via query params or something similar, so that every time we hit this `update` action, we're not changing the status when we didn't mean to.